### PR TITLE
Elaborate on harassment directly in the Code of Conduct

### LINKED ISSUE #152: Citizen Code of Conduct link is dead
The WHATWG Code of Conduct links to https://citizencodeofconduct.org/. That website is down, and it seems like it's down permanently. The [latest snapshot on the Wayback Machine](https://web.archive.org/web/2020*/https://citizencodeofconduct.org/) is about 9 months old, and I found [an issue](https://github.com/stumpsyn/policies/issues/29) that implies that this isn't a temporary outage.

### PR DESCRIPTION
**Pull request by annevk**
Closes #152.

### CODE REVIEW

File: Code of Conduct.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ Contact: the WHATWG Steering Group, via <sg@whatwg.org>, or a member of the comm
...
-* You shall not insult, demean, or harass anyone. We interpret the term "harassment" as including the definition in the [Citizen Code of Conduct](http://citizencodeofconduct.org/); if you have any lack of clarity about what might be included in that concept, please read their definition. In particular, we don't tolerate behavior that excludes people in socially marginalized groups.
+* You shall not insult, demean, or harass anyone. Harassment includes, but is not limited to, offensive verbal comments related to gender, gender identity and expression, age, sexual orientation, disability, physical appearance, body size, race, ethnicity, religion, and technology choices; sexual images in public spaces, deliberate intimidation, stalking, doxing, following, inappropriate photography or recording, inappropriate physical contact, unwelcome sexual attention, sustained disruption of events, as well as advocating for or encouraging harassment. In particular, we don't tolerate behavior that excludes people in socially marginalized groups.
```
Comment (yoavweiss) [inline Code of Conduct.md:10]: Personally, I'd appreciate "nationality" also being on that list

Comment (foolip) [reply Code of Conduct.md:10]: SGTM. @annevk do you want to pick a place in the list to insert it?

Comment (annevk) [reply Code of Conduct.md]: Done. I did notice that we end up saying the same thing twice, sorta, but it's the kind of thing that's worth repeating.

**Review by foolip — APPROVED** [review]

**Review by travisleithead — APPROVED** [review]
Good updates. Nothing further to add.

**Review by othermaciej — APPROVED** [review]

**Pull request merged**